tests/test_opds.py

```diff
@@ -165,9 +165,8 @@ def test__hyperlink_args(self):
         eq_(dict(href=hyperlink.href, rel=hyperlink.rel), m(hyperlink))
 
         # Create a Validation.
-        validation, is_new = get_one_or_create(
-            self._db, Validation, resource=hyperlink.resource
-        )
+        validation, is_new = create(self._db, Validation)
+        hyperlink.resource.validation = validation
```

Comment:
Here's another case where we were using get_one_or_create in a situation that was (sometimes) a little too tricky for SQLAlchemy to figure out what we were looking for.